src/Fhir.Anonymizer.Tool/configuration-sample-draft.json

```diff
@@ -0,0 +1,226 @@
+    //{"path": "Contract.topic", "method": "redact"},
+    //{"path": "Contract.term.topic", "method": "redact"},
+    {"path": "Contract.term.text", "method": "redact"},
+    {"path": "Contract.term.offer.answer.value", "method": "redact"},
+    {"path": "Contract.term.offer.text", "method": "redact"},
+    {"path": "Contract.term.offer.linkId", "method": "redact"},
+    {"path": "Contract.term.asset.context.text", "method": "redact"},
+    {"path": "Contract.term.asset.condition", "method": "redact"},
+    {"path": "Contract.term.asset.text", "method": "redact"},
+    {"path": "Contract.term.asset.linkId", "method": "redact"},
+    {"path": "Contract.term.asset.answer", "method": "redact"},
+    //{"path": "Contract.term.asset.valuedItem.entity", "method": "redact"},
+    {"path": "Contract.term.asset.valuedItem.payment", "method": "redact"},
+    {"path": "Contract.term.asset.valuedItem.linkId", "method": "redact"},
+    {"path": "Contract.term.action.linkId", "method": "redact"},
+    {"path": "Contract.term.action.contextLinkId", "method": "redact"},
+    {"path": "Contract.term.action.requesterLinkId", "method": "redact"},
+    {"path": "Contract.term.action.performerLinkId", "method": "redact"},
+    {"path": "Contract.term.action.reason", "method": "redact"},
+    {"path": "Contract.term.action.reasonLinkId", "method": "redact"},
+    {"path": "Contract.term.group", "method": "redact"},
+    {"path": "ExplanationOfBenefit.disposition", "method": "redact"},
+    {"path": "ExplanationOfBenefit.preAuthRef", "method": "redact"},
+    {"path": "ExplanationOfBenefit.supportingInfo.value", "method": "redact"},
+    {"path": "ExplanationOfBenefit.insurance.preAuthRef", "method": "redact"},
+    {"path": "ExplanationOfBenefit.accident.location", "method": "redact"},
+    {"path": "ExplanationOfBenefit.item.location", "method": "redact"},
+    {"path": "ExplanationOfBenefit.addItem.location", "method": "redact"},
+    {"path": "ExplanationOfBenefit.processNote.text", "method": "redact"},
+    {"path": "ExplanationOfBenefit.benefitBalance.name", "method": "redact"},
+    {"path": "ExplanationOfBenefit.benefitBalance.description", "method": "redact"},
+    {"path": "ExplanationOfBenefit.benefitBalance.financial.allowed", "method": "redact"},
+    {"path": "InsurancePlan.name", "method": "redact"},
+    {"path": "InsurancePlan.alias", "method": "redact"},
+    {"path": "InsurancePlan.coverage.benefit.requirement", "method": "redact"},
+    {"path": "InsurancePlan.plan.generalCost.comment", "method": "redact"},
+    {"path": "HealthcareService.comment", "method": "redact"},  ///Newadd 
+    {"path": "HealthcareService.availabilityExceptions", "method": "redact"}, ///Newadd 
+    {"path": "HealthcareService.extraDetails", "method": "redact"}, ///Newadd
+    {"path": "HealthcareService.eligibility.comment", "method": "redact"}, ///Newadd
+    {"path": "HealthcareService.notAvaliable.description", "method": "redact"}, ///Newadd

```

Comment:
Could we move these rules after OrganizationAffiliation (base/entities#1)? Previously these rules are ordered by its category (base/individuals, base/entities#1...). In the future we could make it in alphabet order to make it easier to maintain.

Reply:
Sure